pysal/weights/Wsets.py

```diff
@@ -531,3 +531,326 @@ def w_clip(w1, w2, outSP=True, silent_island_warning=False):
     return wc
 
 
+def w_stack(ws, silent_island_warning=False):
+    '''
+    Generate a weights object, `w`, that stacks every element of `ws`
+    in the passed index 
+    
+    ...
+
+    Arguments
+    ---------
+    ws                      : list
+                              Sequence of `ps.W` objects to be stitched
+    silent_island_warning   : boolean
+                              Switch to turn off (default on) print statements
+                              for every observation with islands
+
+    Returns
+    -------
+    w                       : W
+                              Resulting `ps.W` object
+
+    Notes
+    -----
+    The resulting `w` contains the original indices, converted to strings if
+    necessary and preceded by 'X-', where X is the order of the original `W`
+    object in `ws`.
+
+    IMPORTANT: Weights are copied from the original weights object and do not
```

Comment:
Maybe? I'm not sure if it's that simple, since the [documentation warns about chaining transformations](https://github.com/pysal/pysal/blob/29f85b8c74053d40ea10533094a7ccfce00d8f5f/pysal/weights/weights.py#L752). 

But, practically, I can do the following just fine:
```python
kw0 =  ps.kernelW_from_shapefile(ps.examples.get_path('columbus.shp'))
kw = ps.kernelW_from_shapefile(ps.examples.get_path('columbus.shp'))
kw.transform = 'r'
kw.transform = 'o'
np.testing.assert_allclose(kw0.full()[0] , kw.full()[0])
```